chat.go

```diff
@@ -62,11 +62,17 @@ type ChatCompletionMessage struct {
 	Name string `json:"name,omitempty"`
 
 	FunctionCall *FunctionCall `json:"function_call,omitempty"`
-	ToolCalls    []ToolCall    `json:"tool_calls,omitempty"`
+
+	// For Role=ASSISTANT prompts this may be set to the tool calls generated by the model, such as function calls.
+	ToolCalls []ToolCall `json:"tool_calls,omitempty"`
```

Comment:
Could you please point out this field in the documentation?

Reply:
also, could we please de-capslock the `Role=ASSISTANT` and `Role=TOOL` here? :)

Reply:
Ah yeah good call re: caps (we use SCREAMING enums internally...).  [Here are the docs](https://platform.openai.com/docs/api-reference/chat/create) for CreateChatRequest.messages.tool_calls:
![image](https://github.com/sashabaranov/go-openai/assets/2048/1f8aef70-bbf9-4613-a2b8-7ac7032a6032)